src/Magnum/Audio/Test/BufferALTest.cpp

```diff
@@ -35,18 +39,109 @@ struct BufferALTest: TestSuite::Tester {
 
     void construct();
 
+    void setData();
+    void setLoopPoints();
+    void setLoopSince();
+    void setLoopUntil();
+    void resetLoopPoints();
+
     Context _context;
 };
 
 BufferALTest::BufferALTest() {
-    addTests({&BufferALTest::construct});
+    addTests({&BufferALTest::construct,
+
+              &BufferALTest::setData,
+              &BufferALTest::setLoopPoints,
+              &BufferALTest::setLoopSince,
+              &BufferALTest::setLoopUntil,
+              &BufferALTest::resetLoopPoints});
 }
 
 void BufferALTest::construct() {
     Buffer buf;
     CORRADE_VERIFY(buf.id() != 0);
 }
 
+void BufferALTest::setData() {
+    Buffer buf;
+    constexpr char data[] { 25, 17, 24, 122, 67, 24, 48, 96 };
+
+    buf.setData(BufferFormat::Mono8, data, 22050);
+
+    ALint bufSize;
+    alGetBufferi(buf.id(), AL_SIZE, &bufSize);
+    CORRADE_VERIFY(bufSize == 8);
```

Comment:
`CORRADE_COMPARE(bufSize, 8)` -- prints a much more useful message when the test fails.

Reply:
Changed locally. Marking this as resolved.